service/src/main/kotlin/no/nav/su/se/bakover/service/revurdering/RevurderingServiceImpl.kt

```diff
@@ -302,6 +323,60 @@ internal class RevurderingServiceImpl(
         }
     }
 
+    override fun underkjenn(
+        revurderingId: UUID,
+        attestering: Attestering
+    ): Either<KunneIkkeUnderkjenneRevurdering, UnderkjentRevurdering> {
+        val revurdering = revurderingRepo.hent(revurderingId)
+            ?: return KunneIkkeUnderkjenneRevurdering.FantIkkeRevurdering.left()
+
+        return when (revurdering) {
+            is RevurderingTilAttestering -> {
```

Comment:
Kunne vurdere å bytte til early return her også, slik dere har gjort for `override fun tilAttestering`